PSSwagger/Paths.psm1

```diff
@@ -1284,6 +1301,24 @@ function Convert-GenericTypeToString {
     return "$($Type.FullName.Substring(0, $Type.FullName.IndexOf('`')))[$genericTypeStr]"
 }
 
+
+function Get-PSCommandOutputType {
+    param(
+        [Parameter(Mandatory = $true)]
+        [Type]$Type
+    )
+
+    if (-not $Type.IsGenericType) {
+        return $Type.FullName.Trim('[]')
+    }
+    elseif($Type.ImplementedInterfaces -and ($Type.ImplementedInterfaces.Name -contains 'IList')) {
```

Comment:
>($Type.ImplementedInterfaces -and ($Type.ImplementedInterfaces.Name -contains 'IList')) [](start = 10, length = 87)

When does this case get hit? Trying to understand if the only cases are (A) non-generic, (B) IList, (C) generic. i.e. are Array, IEnumerable, etc. possible?

Reply:
This case is to handle the scenarios where swagger operation return type is list. 
We are getting the type of GenericTypeArgument of List generic type.
For example: List<String> will be mapped to 'string' as output of the PowerShell cmdlet.


---
In reply to: [141691748](https://github.com/PowerShell/PSSwagger/pull/342#discussion_r141691748) [](ancestors = 141691748)

Reply:
What scenarios are those? When is the output a list instead of an array or any other type of collection object?

---
In reply to: [141702548](https://github.com/PowerShell/PSSwagger/pull/342#discussion_r141702548) [](ancestors = 141702548,141691748)

Reply:
Removed.

---
In reply to: [141703057](https://github.com/PowerShell/PSSwagger/pull/342#discussion_r141703057) [](ancestors = 141703057,141702548,141691748)